OpenRA.Mods.Common/Graphics/ContrailRenderable.cs

```diff
@@ -76,14 +82,22 @@ public void Render(WorldRenderer wr)
 			var screenWidth = wr.ScreenVector(new WVec(1, 0, 0))[0];
 			var wcr = Game.Renderer.WorldRgbaColorRenderer;
 
+			var startColor = startcolor;
```

Comment:
Give this a distinct name. Symbols that differ in only in case are a recipe for confusion. Same with the end logic.

Reply:
converted it to camel case